### ISSUE TITLE
Uncaught Error: ENOTDIR: not a directory, scandir '$ATOM_HOME/.apm\anonymous-cli-metrics.json'

### ISSUE THREAD

**Issue by Zoddo** [idleberg/atom-trashman#1]
[Enter steps to reproduce:]

1. Launch Trashman trough the command palette

**Atom**: 1.39.1 x64
**Electron**: 3.1.10
**OS**: Windows 10 x64 1903
**Thrown From**: [trashman](https://github.com/idleberg/atom-trashman) package 0.1.3

### Stack Trace

Uncaught Error: ENOTDIR: not a directory, scandir 'C:\Users\yprod\.atom\.apm\anonymous-cli-metrics.json'

```
At fs.js:143

Error: ENOTDIR: not a directory, scandir 'C:\Users\yprod\.atom\.apm\anonymous-cli-metrics.json'
    at fs.readdirSync (fs.js:893:3)
    at fs.readdirSync (ELECTRON_ASAR.js:633:28)
    at Object.deleteVersions (/packages/trashman/lib/trashman.coffee:92:16)
    at Object.deleteOutdatedData (/packages/trashman/lib/trashman.coffee:79:10)
    at /packages/trashman/lib/trashman.coffee:55:105)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:348736)
    at Object.didConfirmSelection (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:698372)
    at SelectListView.confirmSelection (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:872494)
    at HTMLDivElement.core:confirm (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:867373)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:348736)
    at KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:1248155)
    at KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:1244289)
    at WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.39.1/resources/app/static/<embedded>:11:283942)
```

### Commands

```
     -1:37 bracket-matcher:remove-brackets-from-selection (div.package-readme.native-key-bindings)
     -1:35.3.0 command-palette:toggle (div.package-readme.native-key-bindings)
     -1:32 core:confirm (input.hidden-input)
     -1:32 trashman:delete-outdated-package-data (div.package-readme.native-key-bindings)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
atom-discord 2.0.2 
autoclose-html 0.23.0 
autocomplete-modules 2.3.0 
git-control 0.9.0 
git-log 0.4.1 
git-plus 8.7.1 
lipsum 0.1.0 
navigate 0.1.20 
node-debug 0.1.7 
node-requirer 0.1.12 
node-resolver2 2.0.2 
php-twig 4.0.0 
pretty-json 2.1.2 
teletype 0.13.4 
termination 0.7.3 
trashman 0.1.3 
```

**Comment by idleberg** [comment]
Thanks for reporting the issue. It should have been fixed in `v0.2.0`!

**Comment by Zoddo** [comment]
Yes, it's fixed! 😉
